apps/new/widget/page/projects/Editor.jsx

```diff
@@ -661,7 +837,7 @@ const SecondScreen = () => {
             src="${alias_old}/widget/components.UploadField"
             props={{
               image: avatar,
-              onChange: (image) => setAvatar({ image }),
+              onChange: (image) => setAvatar(image),
```

Comment:
@elliotBraem I rebased my branch, and get this change, don't really know why it's here, is this the right one?

Reply:
Yes this should be here -- this was the cause for issue why "create new project" wouldn't populate image metadata, it was contructing the wrong object